app/src/main/java/com/yacgroup/yacguide/database/tourbook/TourbookExporter.kt

```diff
@@ -146,28 +146,17 @@ class TourbookExporter(
         val jsonAscends = JSONArray(jsonString)
         _db.deleteAscends()
         _db.deletePartners()
-        val ascends = mutableListOf<Ascend>()
+        val ascents = mutableListOf<Ascend>()
```

Comment:
I think the class `Ascend` also needs to be renamed to `Ascent`. But maybe in a separate PR.

Reply:
I know. This is part of a refactoring issue (#262). However, I believe, renaming this class is not possible without database structure migration (because it is the name of a database entity).